queues.md

```diff
@@ -1223,6 +1224,19 @@ For convenience, you may choose to automatically delete jobs with missing models
      */
     public $deleteWhenMissingModels = true;
 
+<a name="clearing-jobs-from-queues"></a>
+## Clearing Jobs From Queues
+
+If you would like to delete all jobs from the default queue of the default connection, you may do so using the `queue:clear` Artisan command:
+
+    php artisan queue:clear
+
+You may also provide the `connection` argument and `queue` option to delete jobs from a non-default connection or a non-default queue:
+
+    php artisan queue:clear redis --queue=emails
+
+> {note} Clearing jobs from queues is only available for the Redis and Database queue drivers.
+
```

Comment:
It's now available for SQS as well (https://github.com/laravel/framework/pull/34383)

Reply:
Thanks @themsaid, I've updated the PR to include SQS as well!